pages/common/smartctl.md

```diff
@@ -0,0 +1,24 @@
+# smartctl
+
+> View a disk's SMART data and other information.
+> See https://en.wikipedia.org/wiki/S.M.A.R.T for more information.
+
+- View SMART health summary:
+
+`sudo smartctl --health {{/dev/sda}}`
+
+- View device information:
+
+`sudo smartctl --info {{/dev/sda}}`
+
+- Begin a short self-test:
```

Comment:
So this doesn't actually output the result of the short self-test?

Reply:
Self tests run in the background, this command exits after starting the test.

There are tests that run in the foreground, but that pauses all other disk operations, which isn't a good idea if testing a mounted drive.

Check the results is shown in the next commands.

Reply:
Ah, ok. That's ok then - just wanted to check :-)